Fix "ca-certificates-java" issue

https://github.com/lotharschulz/armstrongNumbers/actions/runs/5051449103/job/13675314850 reads:

> Run cd scala
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  ca-certificates-java
0 upgraded, 1 newly installed, 0 to remove and 9 not upgraded.
Need to get [12](https://github.com/lotharschulz/armstrongNumbers/actions/runs/5051449103/job/13675314850#step:4:13).0 kB of archives.
After this operation, 43.0 kB of additional disk space will be used.
Err:1 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 ca-certificates-java all 20190909ubuntu1.1
  404  Not Found [IP: 40.81.[13](https://github.com/lotharschulz/armstrongNumbers/actions/runs/5051449103/job/13675314850#step:4:14).82 80]
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/c/ca-certificates-java/ca-certificates-java_20[19](https://github.com/lotharschulz/armstrongNumbers/actions/runs/5051449103/job/13675314850#step:4:20)0909ubuntu1.1_all.deb  404  Not Found [IP: 40.81.13.82 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.

- [ ] fix it

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -50,8 +50,6 @@ jobs:
     - name: test
       run: |
         cd scala
-        # https://github.com/sbt/sbt/issues/2295#issuecomment-162546065
-        sudo apt-get install ca-certificates-java
         sbt -v +test
 
   python:

